### PR TITLE
Issue #15214: Migrated section 4.8.8 Numeric Literals to whole config testing

### PR DESCRIPTION
**Pull request by Zopsss**
#15214

### CODE REVIEW

File: src/it/resources/com/google/checkstyle/test/chapter4formatting/rule488numericliterals/InputNumericLiterals.java [hunk 1]
```diff
@@ -74,18 +74,18 @@ private long foo() {
...
                   long bad =
                           (4 + 5 * 7 ^ 66l / 7 + 890) // violation 'Should use uppercase 'L'.'
                                   & (88l + 78 * 4); // violation 'Should use uppercase 'L'.'
                   long good = (4 + 5 * 7 ^ 66L / 7 + 890) & (88L + 78 * 4); // ok
                   long[] array = {
-                          66l, // violation 'Should use uppercase 'L'.'
-                          66L, // ok
+                      66l, // violation 'Should use uppercase 'L'.'
+                      66L, // ok
                   };
```
Comment (Zopsss) [inline src/it/resources/com/google/checkstyle/test/chapter4formatting/rule488numericliterals/InputNumericLiterals.java:85]: Formatter didn't intended the code properly, another bug found in the formatter

Comment (romani) [reply src/it/resources/com/google/checkstyle/test/chapter4formatting/rule488numericliterals/InputNumericLiterals.java:85]: Please find existing issue or create new , they are very responsive.

**Review by romani — APPROVED** [review]
Ok to merge

**Review by rdiachenko — APPROVED** [review]
@Zopsss we need to connect the [bug you found](https://github.com/checkstyle/checkstyle/pull/15347/files#r1688494233) with a relevant issue. Please comment with the link to the issue regarding indentation and I'm ok to merge after that.

**Pull request merged**